### PR TITLE
dependencies: include optfeature

### PR DESCRIPTION
**Pull request by juippis**

### CODE REVIEW

**Review by mgorny — CHANGES_REQUESTED** [review]
If there is no preference, then you can't say something is preferred.

**Comment by juippis** [comment]
Switched to *suggested*.

**Comment by mgorny** [comment]
It's still the same deal. Either we don't have a recommendation or we do. I've copied the paragraph on not having a recommendation from wiki page. If we believe we should recommend one way or another, please make it say so explicitly. Then we can vote on it as policy change.

**Comment by juippis** [comment]
Updated to a very simple form honoring original wording.

File: dependencies.rst [hunk 1]
```diff
@@ -15,9 +15,10 @@ acceptable except under very specific circumstances, such as a package
 being nonfunctional unless at least one of a set of optional runtime
 dependencies is installed.
 
-There is no specific preference as to how user should be informed
-of optional runtime dependencies.  The two possible options include
-using ``elog`` messages and ``readme.gentoo-r1`` eclass.
+There is no specific preference as to how user should be informed 
+of optional runtime dependencies. Three possible ways are
```
Comment (mgorny) [inline dependencies.rst]:
```suggestion
of optional runtime dependencies.  Three possible ways are 
```

**Review by Zlogene — APPROVED** [review]

**Review by mgorny — APPROVED** [review]

**Review by SoapGentoo — APPROVED** [review]

**Review by ulm — APPROVED** [review]

**Pull request closed without merging**